stage1/main.tf

```diff
@@ -26,23 +26,23 @@ resource "kubernetes_namespace_v1" "dmtr" {
 }
 
 module "aws_elb_controller" {
-  source       = "../../modules/aws/elb-controller/stage1"
-  count        = var.cloud_provider == "aws" ? 1 : 0
+  source       = "../modules/aws/elb-controller/stage1"
+  count        = var.cloud_provider == "aws" || var.cloud_provider == "gcp" ? 1 : 0
   cluster_name = var.aws_eks_cluster_name
 }
 
 module "aws_storage_classes" {
-  source = "../../modules/aws/storage-classes/stage1"
-  count  = var.cloud_provider == "aws" ? 1 : 0
+  source = "../modules/aws/storage-classes/stage1"
+  count  = var.cloud_provider == "aws" || var.cloud_provider == "gcp" ? 1 : 0
```

Comment:
This creates a storage class using an AWS controller and type. We need one for GCP, instead.

```yaml
apiVersion: storage.k8s.io/v1
kind: StorageClass
metadata:
  name: gp
  annotations:
    storageclass.kubernetes.io/is-default-class: true
allowVolumeExpansion: true
parameters:
  type: pd-balanced
provisioner: pd.csi.storage.gke.io
reclaimPolicy: Delete
volumeBindingMode: WaitForFirstConsumer
```